packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Groups/common/GroupForm/GroupForm.tsx

```diff
@@ -601,7 +615,11 @@ const GroupForm = ({
             <CWButton
               type="submit"
               buttonWidth="wide"
-              disabled={isNameTaken}
+              disabled={
+                isNameTaken ||
+                (requirementSubForms.length === 0 &&
+                  allowedAddresses.length === 0)
```

Comment:
same as above, should we check for `allowlistEnabled` here?

Reply:
This one should be fine not to be feature flagged because we don't even allow the user to delete all the requirements subforms on master.